File: TMessagesProj/src/main/java/ir/amin/HaftTeen/vasni/utils/RtlGrid.java
package ir.amin.HaftTeen.vasni.utils;

import android.content.Context;
import android.support.v7.widget.GridLayoutManager;


public class RtlGrid extends GridLayoutManager {


    public RtlGrid(Context context, int spanCount) {
        super(context, spanCount);
    }

    public RtlGrid(Context context, int spanCount, int orientation, boolean reverseLayout) {
        super(context, spanCount, orientation, reverseLayout);
    }

    @Override
    protected boolean isLayoutRTL() {
        return super.isLayoutRTL();
    }
}

File: message-view/src/main/java/me/himanshusoni/chatmessageview/ui/RtlGrid.kt
package me.himanshusoni.chatmessageview.ui

import android.content.Context
import android.support.v7.widget.GridLayoutManager
import android.util.AttributeSet


class RtlGrid : GridLayoutManager {

    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int, defStyleRes: Int) : super(
            context,
            attrs,
            defStyleAttr,
            defStyleRes
    ) {
    }

    constructor(context: Context, spanCount: Int) : super(context, spanCount) {}

    constructor(context: Context, spanCount: Int, orientation: Int, reverseLayout: Boolean) : super(
            context,
            spanCount,
            orientation,
            reverseLayout
    ) {
    }

    override fun isLayoutRTL(): Boolean {
        return true
    }
}